tools/generator/src/Generator+SetTargetConfigurations.swift

```diff
@@ -460,6 +559,17 @@ extension Target {
     }
 }
 
+private extension Platform.OS {
+    var sdkRoot: String {
```

Comment:
Seems like this should be moved to a more central place at some point.

Alternatively, I think you could use XcodeProj for this: https://github.com/tuist/XcodeProj/blob/8c12577a2e07b119c85bbee118e719cc4c9f8af7/Sources/XcodeProj/Utils/BuildSettingsProvider.swift#L46

Reply:
Yeah, I generally move them once I need to remove the `private`.